lib/models/file.js

```diff
@@ -31,11 +31,6 @@ function FileModelFactory (Model) {
                     uuidv4: true
                 },
 
-                md5: {
```

Comment:
This should stay in the model.  on-http is updating the md5sum using crypto (https://github.com/RackHD/on-http/blob/fea2324006a440f2ed8f48972502698f34983de6/lib/services/files/file-plugin.js#L146) and we reference the md5sum in the workflows.